array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-vars */
+function initial(array) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i !== array.length - 1) {
```

Comment:
Instead of continuing until the end and having a conditional to ignore the last item, could you do anything to make the loop never reach that index? Remember that you get to decide where a loop begins and ends when you write it!